src/lib/pipeline_configuration.py

```diff
@@ -30,6 +33,7 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
         self.binary_code_scheme = binary_code_scheme
         self.binary_coded_field = binary_coded_field
         self.binary_analysis_file_key = binary_analysis_file_key
+        self.course_name = course_name
```

Comment:
What is this for?

Reply:
We thought we would need to disaggregate Coda output by course, this was a first step in that direction. We didn't need to. Removed.